mysql-test/collections/disabled.def

```diff
@@ -251,6 +283,67 @@ galera.galera_fk_lock_parent_update_child : BUG#0 Few cases have timing issues(P
 galera.galera_toi_ddl_fk_insert : BUG#0 CODERSHIP qa#39 test fails sporadically (PXC-3431)
 
 
+# encryption
+# Probably we should use upstream version of encryption tests for PXC
+encryption.percona_parallel_dblwr_encrypt                               : BUG#0 PXC does not support KEYRING and ONLINE_TO_KEYRING encyption methods
```

Comment:
It should be safe to remove the encryption testsuite from the defaults for pxc.

Reply:
PXC-3557